### PR TITLE
added element multiplicity for stateful-environment-record

### PR DESCRIPTION
**Pull request by nedmsmith**
Added multiplicity of elements for conditional and conditional series triples via stateful environment record.

Addresses part of the issues raised in #288 

Note: Currently PR only touches CDDL and diag files. A cursory review of the prose sections doesn't appear to require updating except for sections that describe transformation of external to internal representation of the conditional series triple.

### CODE REVIEW

File: cddl/conditional-series-record.cddl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 conditional-series-record = [
   ; reference values to be matched against evidence
-  refv: measurement-values-map
+  refv: measurement-map
```
Comment (deeglaze) [inline cddl/conditional-series-record.cddl:3]: Why is this not [ + measurement-map ] to allow a condition on more than one mkey? I would imagine there are endorsements that are conditional on multiple elements matching expectations and not just one element. Same for adding endorsements.

**Comment by yogeshbdeshpande** [comment]
This requirement folded into PR #276

**Comment by yogeshbdeshpande** [comment]
@nedmsmith we can close this PR, if you think it is appropriate!

**Comment by nedmsmith** [comment]
Ok to close if superseded by your pr

**Comment by yogeshbdeshpande** [comment]
Multiplicity of stateful environments  and modelling conditional endorsement series triple already folded into PR #276

**Pull request closed without merging**